utils/merkle-tree/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name = "beefy-merkle-tree"
+name = "merkle-tree"
```

```suggestion
name = "binary-merkle-tree"
```

Should be more expressive. First time I was thinking on what the difference to our trie is until I read the description.

---

updated